README.md

```diff
@@ -26,7 +26,7 @@ Add this plugin to your gatsby-config.js.
 
 ```
 module.exports = {
-  plugins: ['gatsby-plugin-tailwindcss],
+  plugins: ['gatsby-plugin-tailwindcss]',
```

Comment:
Hey @juankaram , 
Thank you very much for your contribution. Isn't the single quote should be here?

```suggestion
  plugins: ['gatsby-plugin-tailwindcss'],
```

I really appreacite your PR 😄 . Can we fix that, then I'll merge the pull request

Reply:
Absolutely! Thanks for double checking it!